docs/plugins.md

```diff
@@ -229,6 +229,37 @@ class MyCompiler {
 
 Note: exported JS will not be compiled or linter by any other plugin and its `require` statements will not be resolved. Make sure your exported JS is self-contained.
 
+### Static file compilers
```

Comment:
@paulmillr this is a glimpse of what `static-jade-brunch` would look like with this new API